static/app/views/performance/browser/webVitals/pagePerformanceTables.tsx

```diff
@@ -0,0 +1,202 @@
+import {useTheme} from '@emotion/react';
+import styled from '@emotion/styled';
+
+import GridEditable, {
+  COL_WIDTH_UNDEFINED,
+  GridColumnHeader,
+  GridColumnOrder,
+  GridColumnSortBy,
+} from 'sentry/components/gridEditable';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {useDiscoverQuery} from 'sentry/utils/discover/discoverQuery';
+import EventView from 'sentry/utils/discover/eventView';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import {calculatePerformanceScore} from 'sentry/views/performance/browser/webVitals/utils/calculatePerformanceScore';
+import {getScoreColor} from 'sentry/views/performance/browser/webVitals/utils/getScoreColor';
+import {Row} from 'sentry/views/performance/browser/webVitals/utils/types';
+
+type RowWithScore = Row & {score: number};
+
+const MAX_ROWS = 7;
+
+type Column = GridColumnHeader<keyof RowWithScore>;
+
+const columnOrder: GridColumnOrder<keyof RowWithScore>[] = [
+  {key: 'transaction', width: COL_WIDTH_UNDEFINED, name: 'Transaction'},
+  {key: 'count()', width: COL_WIDTH_UNDEFINED, name: 'Count'},
+  {key: 'score', width: COL_WIDTH_UNDEFINED, name: 'Score'},
+];
+
+const sort: GridColumnSortBy<keyof Row> = {key: 'count()', order: 'desc'};
+
+export function PagePerformanceTables() {
+  const theme = useTheme();
+  const organization = useOrganization();
+  const pageFilters = usePageFilters();
+  const location = useLocation();
+  const eventView = EventView.fromNewQueryWithPageFilters(
+    {
+      fields: [
+        'transaction',
+        'transaction.op',
+        'p75(measurements.lcp)',
+        'p75(measurements.fcp)',
+        'p75(measurements.cls)',
+        'p75(measurements.app_init_long_tasks)',
+        'count()',
+      ],
+      name: 'Web Vitals',
+      query:
+        'transaction.op:pageload (transaction:/performance* or transaction:/discover* or transaction:/dashboards*)',
+      orderby: `-count()`,
+      version: 2,
+    },
+    pageFilters.selection
+  );
+  const {data, isLoading} = useDiscoverQuery({
+    eventView,
+    limit: 50,
+    location,
+    orgSlug: organization.slug,
+    options: {
+      enabled: pageFilters.isReady,
+      refetchOnWindowFocus: false,
+    },
+  });
```

Comment:
Long term if bot to have a seperate hook for this data so that it cleans it up and we can introduce some better typing.